src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double base;
+    private final double height;
+
+    public RightTriangle(String color, double base, double height) {
+        super(color);
+        this.base = base;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (height * base) / 2;
+    }
+
+    @Override
+    public void draw() {
```

Comment:
The method implementation should not begin with an empty line. Please remove the empty line above the draw method.